drivers/filesystems/ext2/src/ext3/htree.c

```diff
@@ -1596,6 +1596,11 @@ int ext3_release_dir (struct inode * inode, struct file * filp)
     return 0;
 }
 
+#if defined(__REACTOS__) && defined(__clang__)
+/* FIXME: Inspect */
+struct ext3_dir_entry_2* do_split(struct ext2_icb *icb, struct inode *dir, struct buffer_head **bh,struct dx_frame *frame, struct dx_hash_info *hinfo, int *error);
+#endif
```

Comment:
It is needed here but not allowed in the header?

Reply:
I got a redefinition and I couldn't quite see the difference anywhere, so I assumed some deeper cpp/includes issue => locality.